### ISSUE TITLE
On other database training

### ISSUE THREAD

**Issue by fantasy4335** [thulab/DeepHash#46]
I train DHN with CUB200-2011(fine-grained image), and I correctly list train.txt, test.txt and database.txt. But the result(mAP) is 0.3%, just not train. What is wrong with it?

**Comment by bl0** [comment]
Hi, we have not tried the fine-grained datasets. So I don't have any experience to help. Sorry about that.
In my view, you can try to disable the quantization loss and check the model output, such as the prediction similarity on the continues feature, to see whether the model learns something. If the model learns nothing, there may be something wrong, code or hyperparameter, or maybe the fine-grained dataset is too hard to learn a pairwise metric.
If the model learns the similarity in the train set but performed poorly in the test set, then it's an overfitting problem.

**Comment by bl0** [comment]
I am closing this issue due to a long time of inactivity, feel free to reopen it if there is any question. Thanks!
